backend/services/theme_service.py

```
@@ -51,6 +51,60 @@ def __init__(
+            requested_strength: The requested strength parameter (default 0.4)
+            
+        Returns:
+            float: The final strength to use, controlled by feature flags and safety checks
+            
+        Feature Flag Logic:
+        - AGGRESSIVE_THEME_TESTING=true: Enables controlled aggressive testing (0.6-0.8)
+        - TESTING_MODE=true: Allows higher strength for development testing
+        - Default: Uses safe range (0.3-0.4) for production
+        - A/B Testing: Future enhancement for user subset testing
+        """
+        import os
```

Ruff (0.12.2)

Redefinition of unused `os` from line 11

Remove definition: `os`

(F811)